cfme/utils/dockerbot/sel_container.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/usr/bin/env python
```

Comment:
Q: why some script pointing to `py2` other `py3`? lets put `python` defaults and it will pick up as per environment